listeners/measured.go

```diff
@@ -27,14 +27,14 @@ func (l *stateAwareMeasuredListener) Accept() (c net.Conn, err error) {
 	sac, _ := c.(*measured.Conn).Conn.(WrapConnEmbeddable)
 	return &wrapMeasuredConn{
 		WrapConnEmbeddable: sac,
-		Conn:               *c.(*measured.Conn),
+		Conn:               c.(*measured.Conn),
```

Comment:
We have more listeners using the same pattern. These don't make the race detector complain?